packages/conjure-client/rollup.config.mjs

```diff
@@ -0,0 +1,25 @@
+// rollup.config.js
+import typescript from "@rollup/plugin-typescript";
+import replace from "@rollup/plugin-replace";
+
+export default {
+  input: "src/index.ts",
+  output: {
+    dir: "lib-modern",
+    format: "es",
+    sourcemap: true,
+  },
+  treeshake: {
+    moduleSideEffects: false, // shake away the ponyfill
```

Comment:
Is this piece necessary? I would think that marking the module as external would be sufficient since that import shouldn't be side-effectful (since it's not actually polyfilling globally, just providing a ponyfill implementation)

Reply:
Ah, I guess if we didn't have this then the import would stay there and it'd be expected that the consumer does have the dependency

Reply:
Correct.  Rollup is being careful so we have to tell it there are no side effects.